DCOS/utils/kill-mesos-agent-ungraceful.ps1

```diff
@@ -0,0 +1,48 @@
+$mesosServiceObj = Stop-Service dcos-mesos-slave -PassThru
+$mesosServiceObj.WaitForStatus('Stopped','00:00:30')
+if ($mesosServiceObj.Status -ne 'Stopped') {
+    Write-Output "Failed to stop the service"
+    exit 1
+} else {
```

Comment:
Same comment as above. Remove the `else` branch since it's not needed. You'll exit the script on the `then` branch.